react/CircleButton/helpers.js

```diff
@@ -0,0 +1,19 @@
+export const makeTypoColor = ({ theme, active, disabled }) => {
+  if (disabled) {
+    return theme.palette.text.disabled
+  }
+  if (active) {
+    return theme.palette.primary.main
+  }
+  return theme.palette.text.primary
+}
+
+export const makeButtonShadow = ({ theme, active, disabled }) => {
+  if (disabled) {
+    return `inset 0 0 0 1px ${theme.palette.border.main}`
+  }
+  if (active) {
+    return `none`
+  }
+  return `inset 0 0 0 1px ${theme.palette.border.main}`
```

Comment:
nit:
```
  if (active) {
    return `none`
  }
  return `inset 0 0 0 1px ${theme.palette.border.main}`
  ```

Reply:
no need to keep line 12, 13, 14

Reply:
si, car dans le cas d'un active ET disabled, le disabled doit prendre la main sur le active